sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/ExceptionExtensions.cs

```diff
@@ -33,16 +33,16 @@ internal static class ExceptionExtensions
             switch (instance)
             {
                 case AmqpException amqpEx:
-                    return AmqpError.CreateExceptionForError(amqpEx.Error, eventHubName);
+                    return AmqpError.CreateExceptionForError(amqpEx.Error, eventHubName, instance);
```

Comment:
Just curious why we use `instance` here and the case filtered exception below?

Reply:
There's no particular reason;  I'll adjust to keep things consistent.